src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -732,4 +732,137 @@ public override TransformWrapper Fit(IDataView input)
             return new TransformWrapper(Host, new NgramHashTransform(Host, args, input));
         }
     }
+
+    /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
+    public sealed class LdaEstimator : TrainedWrapperEstimatorBase
+    {
+        private readonly (string input, string output)[] _columns;
+        private readonly int _numTopic;
+        private readonly float _alphaSum;
+        private readonly Single _beta;
+        private readonly int _mhstep;
+        private readonly int _numIterations;
+        private readonly int _likelihoodInterval;
+        private readonly int _numMaxDocToken;
+        private readonly int? _numThreads;
+        private readonly int _numSummaryTermPerTopic;
+        private readonly int _numBurninIterations;
+        private readonly bool _resetRandomGenerator;
+        private readonly bool _outputTopicWordSummary;
+
+        /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
+        /// <param name="env">The environment.</param>
```

Comment:
>param [](start = 13, length = 5)

One potential pattern you can also use (for example, if there is so many arguments) is to have a single `Action<Arguments> advancedSettings` parameter (check FastTree estimators for examples of that).

Reply:
This seems to be a case where you would benefit from it. Like pretty much all parameters except `numTopic` would be better off in advanced.

---
In reply to: [219620113](https://github.com/dotnet/machinelearning/pull/972#discussion_r219620113) [](ancestors = 219620113)